packages/charts-bar/src/charts-bar.scss

```diff
@@ -0,0 +1,61 @@
+// scss-lint:disable all
+
+.ibm-chart {
+
+  @import "https://raw.githubusercontent.com/IBM-Design/colors/master/_ibm-colors.scss";
```

Comment:
Definitely... This comes directly from GitHub but I understand we should change this very soon.